Nos/Models/CoreData/Event+CoreDataClass.swift

```diff
@@ -1088,7 +1088,6 @@ public class Event: NosManagedObject, VerifiableEvent {
                     deletedEvent.deletedOn.insert(relay)
                 }
             }
-            try context.saveIfNeeded()
```

Comment:
This one weird trick helps the app church through events a lot quicker. The parse loop in RelayService already calls save after each batch of events is processed, so this is superfluous.